app/service/ClaimAmountService.scala

```diff
@@ -78,4 +82,13 @@ class ClaimAmountService @Inject()(
         Seq(standardRate(claimAmount), scottishRate(claimAmount))
     }
   }
+
+  def filterRecords(taxCodeRecord: Seq[TaxCodeRecord]): Option[TaxCodeRecord] = {
+    taxCodeRecord.find(_.status == Live) match {
+      case Some(liveTaxCodeRecord) => Some(liveTaxCodeRecord)
+      case None if taxCodeRecord.nonEmpty => taxCodeRecord.headOption
+      case None if taxCodeRecord.isEmpty => None
+    }
```

Comment:
What happens if there are multiple records with status == Live?

Reply:
It will return the first record it has found that is live

Reply:
Or multiple Live with a mixture of Scottish and English tax codes?